pkg/og/convert/pprof/profile.go

```diff
@@ -62,6 +62,8 @@ func (p *RawProfile) ParseToPprof(_ context.Context, md ingestion.Metadata) (res
 
 	fixTime(profile, md)
 	fixFunctionNamesForScriptingLanguages(profile, md)
+	fixFunctionIDForBrokenDotnet(profile, md)
```

Comment:
let's add this only for spyName == "dotnetspy"
this will require to update our examples as they are old as hell and has no spyName in the request query